assistant/src/main/java/com/ibm/watson/assistant/v1/model/CreateDialogNodeOptions.java

```diff
@@ -120,6 +120,7 @@
   private String digressOut;
   private String digressOutSlots;
   private String userLabel;
+  private Boolean disambiguationOptOut;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/watson-developer-cloud/java-sdk/pullRequest?prid=4552601)